ci-operator/step-registry/hypershift/aws/setup-nested-management-cluster/hypershift-aws-setup-nested-management-cluster-chain.yaml

```diff
@@ -7,6 +7,9 @@ chain:
     - name: HYPERSHIFT_NODE_COUNT
       default: "3"
       documentation: "The number nodes for the cluster. This number will get mulitplied by three, as there are three AZs"
+    - name: INSTANCE_TYPE
```

Comment:
it'd be good to prefix with HYPERSHIFT_ to avoid potential clashes with ci built in env vars, e.g. that happened with namespace